client/src/components/main/navigation/forms/EditSystemNotificationForm.js

```diff
@@ -125,13 +125,13 @@ export default class EditSystemNotificationForm extends React.Component {
               ? this.props.notification.severity : 'INFO'}`
           })(
           <Select>
-            <Select.Option key="INFO" value="INFO">
+            <Select.Option key="INFO" value="INFO" title={"Info"}>
```

Comment:
There is no need to use braces for constant strings, i.e. `title={"Info"}` should be replaced with `title="Info"`.
The same is for `warning` and `critical` options